loadrunner_professional/README.md

```diff
@@ -0,0 +1,58 @@
+# LoadRunner Professional
+
+## Overview
+
+OpenText™ LoadRunner Professional provides a load testing solution that enables you to test the performance of diverse application types, and to identify and resolve issues before applications go live.
+
+LoadRunner Controller is a tool for creating and controlling LoadRunner Professional scenarios. A scenario defines the events that occur during each testing session. It controls the number of users to emulate (virtual users, or Vusers), the actions they perform, and the machines on which they run their emulations. You use scenarios to create load tests to check the reliability and strength of your servers.
+
+You can configure Controller to push real-time metrics and data from scenario runs to Datadog.
+
+
+## Setup
+
+Configure LoadRunner Controller to push data to Datadog. You can choose whether to send scenario information and/or run metrics. In Datadog, you can install a dashboard to view the data in pre-configured widgets.
+
+1.	In the Controller toolbar, select __Tools > Datadog Configuration__.
+2.	In the __Site__ field, select the region defined in your Datadog account.
+3.	In the __API Key__ field, enter the API key generated by Datadog.
+4.	Click __Test Connection__.
+5.	Once the connection is successful, select whether to send scenario information and/or run information to Datadog:
+
+|   |   |
+|---|---|
+|__Send scenario information__| Send information about the scenario run, such as the start and stop time, duration, and included scripts. |
+|__Send run information__| Send metrics from the scenario run, such as Vuser status and transaction response times. |
+	
+6.	If you enable Controller to send scenario information, install the __LoadRunner Run Info Pipeline__ in Datadog to ensure that tags are applied to the scenario information.
```

Comment:
How does a user install the LoadRunner Run Info Pipeline?

Reply:
I am refering to the LoadRunner log pipeline that will be available in DataDog here. Maybe put the link form Datadog Docs on how to install a log pipeline?

Reply:
Ah i see - updated my suggestion below